portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,71 @@
-#content {
+body { /*page background*/
```

Comment:
It's great that you added a bunch more styling to your portfolio!

I highly recommend breaking up changes that are independent (functionality, styling, an implementation of a class, etc.) to be broken up into multiple PRs. I'll go ahead and approve this one, but for future PRs, we should expect to see each PR correspond to a single logical unit. See the guidelines I put in our process doc for the reasons on why large PRs are a bad idea and how large in general PRs should be.